engines/python/setup/djl_python/huggingface.py

```diff
@@ -540,6 +540,36 @@ def _read_model_config(self, model_config_path: str):
                 f"This is required for loading huggingface models")
             raise e
 
+    def quantize(self, properties: dict):
```

Comment:
why we add this in HF.py? We don't offer JIT for AWQ model conversion for vLLM

Reply:
Should I instead create a new handler that is only used for AOT? The benefit with the HF.py handler is being able to use the model/property initialization methods offered with the HuggingFaceService.

Reply:
Had a discussion with Frank and Qing regarding quantize or partition. Your design is very similar to partition which is fine, but the user can bring in their own partition  script, but  we dont want to make quantize entryPoint public like that. So let's remove quantize from `huggingface.py `and move to `partition.py`. It does not have to call any handler at all. In this case you dont need `run_partition.py` script at all.

Just make sure, you have all the dependencies your partition script. (python engine jar extraction implementation is already there, so you should be fine). You dont have to use HuggingFaceService at all, move `_init_model` and `_init_tokenizer` `utils.py` . You can still make use of HuggingFaceProperties. 

Or if you don't want to do the refactoring, leave a TODO and write your own initialization of model and tokenizer, we can take of the refactoring later.